common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -508,3 +508,65 @@ def test_studio_settings_page_a11y(self):
         })
 
         self.settings_page.a11y_audit.check_for_accessibility_errors()
+
+
+class AdvancedSettingsTest(StudioCourseTest):
+    """
+    Tests for Advanced Settings tab in studio.
+    """
+    def setUp(self):  # pylint: disable=arguments-differ
+        super(AdvancedSettingsTest, self).setUp()
+        self.advanced_settings = AdvancedSettingsPage(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+        )
+
+        self.type_fields = ['Course Display Name', 'Advanced Module List', 'Discussion Topic Mapping',
+                            'Maximum Attempts', 'Course Announcement Date']
+
+        # Before every test, make sure to visit the page first
+        self.advanced_settings.visit()
+        self.assertTrue(self.advanced_settings.is_browser_on_page())
```

Comment:
You can remove this line. `.visit()` will call `is_browser_on_page` implicitly. It will wait for `is_browser_on_page` to be True and then will fail if it doesn't become true within the timeout period (30 seconds).